dayforce_client/client.py

```diff
@@ -99,17 +103,68 @@ class DayforceSFTP(object):
     username: str = attr.ib()
     password: str = attr.ib(repr=False)
     port: int = attr.ib(default=22)
-    known_hosts: str = attr.ib(default='~/.ssh/known_hosts')
+    # the host key can be retrieved with ssh-keyscan (pass just the base64 encoded part of the line beginning with ssh-rsa)
+    host_key: str = attr.ib(default=None)
     disable_host_key_checking: bool = attr.ib(default=False)
 
+    _sftp: pysftp.Connection = None
+    _sftp_live = bool = False
```

Comment:
@bobzoller I think these need to be `attr.ib()`'s, yeah? So something like:

```python
    _sftp: Optional[pysftp.Connection] = attr.ib(default=None)
    _sftp_live: bool = attr.ib(default=False)
```

Reply:
awesome, thanks.  I wasn't sure if I should be using attr for private things or not